src/test/java/org/mongojack/TestDBUpdateSerialization.java

```diff
@@ -20,6 +20,7 @@
 import static org.hamcrest.core.IsNull.notNullValue;
 import static org.hamcrest.core.IsNull.nullValue;
 import static org.junit.Assert.assertThat;
+import static org.mongojack.TestDBUpdateSerialization.NestedIdFieldWithDifferentType.NESTED_ID_FIELD_VALUE;
```

Comment:
is this import really needed? we're already in the TestDBUpdateSerialization file, so it's surprising to me to see this